### PR TITLE
RTK.IO Aardvark Header Bidding Adapter

### PR DESCRIPTION
**Pull request by naegelin**
RTK.IO's aardvark service provides an http header bidding endpoint for RTK.IO direct demand sources.

### CODE REVIEW

File: src/adapters/aardvark.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+  function _callBids(params) {
+    var rtkBids = params.bids || [];
+
+    _requestBids(rtkBids);
+  }
+
+  function _requestBids(bidReqs) {
+    // build bid request object
+    var ref = window.top.location.host;
```
Comment (mkendall07) [inline src/adapters/aardvark.js]: please put this in try/catch in case inaccessible.

File: integrationExamples/gpt/pbjs_example_gpt.html [hunk 2]
```diff
@@ -82,6 +82,12 @@
                             placementId: 'TO ADD'
                         }
                     }, {
+                        bidder:"aardvark",
+                        params: {
+                            ai: "TO ADD",
```
Comment (mkendall07) [inline integrationExamples/gpt/pbjs_example_gpt.html]: please add comments here on what the params mean

File: src/adapters/aardvark.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            bidResponse.height = currentBid.sizes[0][1];
+            returnedBidIDs[bid.id] = currentBid.placementCode;
+            bidmanager.addBidResponse(currentBid.placementCode, bidResponse);
+          }
+
+        }
+
+      });
+
+    }
```
Comment (mkendall07) [inline src/adapters/aardvark.js]: Please create an `else` condition here to add a `no bid` type response. Example `bidfactory.createBid(constants.STATUS.NO_BID)`

File: src/adapters/aardvark.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    var scURL = "";
+
+    if (shortcodes.length > 1) {
+      scURL = shortcodes.join("_");
+    } else {
+      scURL = shortcodes[0];
+    }
+
+    var scriptUrl = '//thor.rtk.io/' + ai + "/" + scURL + "/aardvark/?jsonp=window.pbjs.aardvarkResponse&rtkreferer=" + ref;
+    adloader.loadScript(scriptUrl, null);
```
Comment (mkendall07) [inline src/adapters/aardvark.js]: no need to pass `null`

**Comment by naegelin** [comment]
Thanks for the review. Requested changes have been checked in.

**Comment by mkendall07** [comment]
@naegelin 
Thanks. Can you please provide use with test params to validate a bid response? Once confirmed we can merge.

**Comment by naegelin** [comment]
`
params: {
  "ai": "Ohot",
  "sc": "sXvU",
},`

**Comment by naegelin** [comment]
@mkendall07 let me know if you need anything else.

**Comment by matthewlane** [comment]
@naegelin Not getting bid responses with the provided params, are those still good for testing?

**Comment by naegelin** [comment]
Sorry about that. re-enabled them. try again please :)

**Comment by matthewlane** [comment]
Getting them now, thanks @naegelin!

**Pull request merged**